### ISSUE TITLE
Producing a .img file instead of flashing

### ISSUE THREAD

**Issue by faddat** [pikvm/pi-builder#2]
I am testing out this system for use in [Starport](https://github.com/tendermint/starport), a rapid blockchain development tool that allows developers and entrepreneurs to ship blockchain products as hardware.

I am trying to figure out if there is a way to get output as an image file.

Currently, I have been using [packer-builder-arm](https://github.com/mkaczanowski/packer-builder-arm) but the docker-based approach used here looks interesting.  

# Goal

Basically, I just need to know how to build an image file instead of flashing to SD card, and we could use pi-builder and significantly lower our build times and increase development speed by reducing the length of CI cycles.

**Comment by mdevaev** [comment]
Hello. I see what you want. Try something like this: https://github.com/pikvm/os/blob/master/Makefile#L106
I use this approach and it works fine for me.

**Comment by faddat** [comment]
Thank you, I saw that yesterday.  

I am really looking forward to trying this out-- I did some early builds and this will likely take our image build time down dramatically because of the docker cache 🙏 .

I often write Github actions, would you like one for your OS image?  Won't be much different from the one I make later today.

**Comment by mdevaev** [comment]
Thanks, but the build is usually done on my local build farm, so I don't want to waste time on something I won't use (at least not right now). If I need it, could I ask you later? :)
